### PR TITLE
Update docs/quickstart-guide.md

### PR DESCRIPTION
**Pull request by tmrekk121**
Update QuickStart to a format using an all-in manifest

### CODE REVIEW

File: docs/quickstart-guide.md [hunk 1]
```diff
@@ -2,147 +2,140 @@
...
+secret/gatling-notification-slack-secrets unchanged
+gatling.gatling-operator.tech.zozo.com/gatling-sample01 created
 ```
 
-## Running your first load testing
+After execution, the ServiceAccount and Gatling CR required to run the Gatling Runner Pod will be deployed.
 
-Sample Gatling load testing data is provided in the gatling folder.
-Just like you do for the Operator, you can proceed all steps needed for deploying sample Gatling CR with GNU make.
+After deploying the Gatling CR, the Gatling CR, Gatling Runner Job, and Gatling Runner Pod will be generated and the Gatling test scenario will be executed.
```
Comment (yokawasa) [inline docs/quickstart-guide.md:59]: How about this?
```suggestion
After deploying the Gatling CR, Gatling Controller creates a Gatling Runner job. The job then runs the Gatling Runner pods to execute Gatling load test scenario in parallel.
```

File: docs/quickstart-guide.md [hunk 2]
```diff
@@ -2,147 +2,140 @@
...
 
-### Deploying and running the load testing locally
+```
+$ kubectl get gatling,job,pod
+```
 
-To deploy sample Gatling CR to a local Kubernetes cluster/Kind instance:
+The output is similar to:
```
Comment (yokawasa) [inline docs/quickstart-guide.md:65]:
```suggestion
Expected output would be like this:
```

File: docs/quickstart-guide.md [hunk 3]
```diff
@@ -2,147 +2,140 @@
...
-```bash
-make sample-docker-push SAMPLE_IMG=<your-registry>/gatling:<tag>
+```
+kubectl logs gatling-sample01-runner-tkplh -c gatling-runner -f
 ```
 
-> 📝  Ensure that you're logged into your docker container registry that you will be using as the image store for your K8s cluster if not yet done!
-
-After you push the sample gatling container, deploy the Gatling CR to your cluster:
+The output is similar to:
```
Comment (yokawasa) [inline docs/quickstart-guide.md:86]:
```suggestion
Expected output would be like this:
```

File: docs/quickstart-guide.md [hunk 4]
```diff
@@ -2,147 +2,140 @@
...
+---- Scenario Name -------------------------------------------------------------
+[##########################################################################]100%
+          waiting: 0      / active: 0      / done: 1
+================================================================================
+
+Simulation MyBasicSimulation completed in 14 seconds
 ```
 
-> :memo: Ensure you're connected to your K8s cluster
+In this example, the notification of Gatling result reports and the storage of the result reports in the cloud provider are not performed.
```
Comment (yokawasa) [inline docs/quickstart-guide.md:139]: Here is my suggestion for this
```suggestion
As configured in [the sample manifest](https://github.com/st-tech/gatling-operator/blob/85e69840274214c47e63f65a5c807dd541dff245/config/samples/gatling-operator_v1alpha1_gatling01.yaml#L6-L8), an aggregated Gatling HTML report is not created, nor notification result message is posted.
```

File: docs/quickstart-guide.md [hunk 5]
```diff
@@ -2,147 +2,140 @@
...
 
-> :memo: Ensure your cluster has permissions to pull containers from your container registry
+This can be done by setting the .spec.cloudStorageSpec and .spec.notificationServiceSpec.
```
Comment (yokawasa) [inline docs/quickstart-guide.md:141]: Here is my suggestion for this
```suggestion
You can generate the Gatling HTML report by enabling `.spec.generateReport` flag and setting the `.spec.cloudStorageSpec`. Also you can posting the notification message by enabling `.spec. notifyReport ` and setting `.spec.notificationServiceSpec`. For more information about configuring Gatling CR, please refer to [User Guide](./user-guide.md)
```

**Review by yokawasa — COMMENTED** [review]
I've add comments for sentences that I've noticed a little modification needed. Please check

**Review by tmrekk121 — COMMENTED** [review]
@yokawasa 
Thanks for the suggestion.
Fixed it!

**Review by yokawasa — APPROVED** [review]
@tmrekk121 
lgtm!

I think it would be great to have another sample Gatling CR running where both reporting and notification are enabled. We can add that to the quickstart guide in another PR

**Pull request merged**